File: code/97245.pas
var s : string;
    n, dem, smax, i, j, v : integer;
BEGIN
    readln(s);
    n:=length(s);
    smax:=0;
    for i:= 1 to n do begin
        dem:=0;
        for j:= 1 to n do 
            if s[i]=s[j] then inc(dem);
        if dem > smax then begin
            smax:=dem;
            v:=i;
        end;
    end;
    write(s[v],' ',smax)
END.

File: code/97245.py
s=input()
n=len(s)
smax=0
for i in range(n):
    dem=0
    for j in range(n):
        if s[i]==s[j]: dem+=1
    if dem>smax:
        smax=dem
        v=i
print(s[v],smax)